SportsHub/SportsHub.Api/Exceptions/CustomExceptionModels/CustomException.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace SportsHub.Api.Exceptions.CustomExceptionModels;
+
+public abstract class CustomException: Exception
+{
+    public int StatusCode { get; set; }
```

Comment:
I think here better to keep an ErrorCode, not status

Reply:
Add the possibility for the business logic exception, for example, to be more generic. Allow setting not only a message but an error code as well

Reply:
Okey i will create additional constructor for this